gym/core.py

```diff
@@ -142,64 +156,66 @@ def render(self, mode="human"):
               the list of supported modes. It's recommended to call super()
               in implementations to use the functionality of this method.
 
-        Args:
-            mode (str): the mode to render with
-
-        Example::
-
-            class MyEnv(Env):
-                metadata = {'render_modes': ['human', 'rgb_array']}
+        Example:
+            >>> class MyEnv(Env):
+            ...    metadata = {'render_modes': ['human', 'rgb_array']}
+            ...
+            ...    def render(self, mode='human'):
+            ...        if mode == 'rgb_array':
+            ...            return np.array(...) # return RGB frame suitable for video
+            ...        elif mode == 'human':
+            ...            ... # pop up a window and render
+            ...        else:
+            ...            super(MyEnv, self).render(mode=mode) # just raise an exception
```

Comment:
Why is this in interactive interpreter mode? I see how that makes sense for one-line stuff but you don't define entire functions in interactive mode

Reply:
Personally, I have been making all code examples in an interactive interpreter style. 
Otherwise, Im unsure when to have `>>>` and `...`. How would you do it? Im happy to change it to that

Reply:
I like to use interactive style if it is a one-liner that produces some output that is of interest. E.g. if we sample from a `Space`: It can be done in a single line and the output is actually important for understanding what the method does. 

For blocks that don't produce any output, I don't see any advantage in using interactive style (except that it magically enables syntax highlighting in PyCharm for some reason). But that's just my taste

Reply:
Fair, that makes sense. I think that main reason I did this was for pycharm